pandas/core/internals/blocks.py

```diff
@@ -319,10 +312,7 @@ def getitem_block(self, slicer, new_mgr_locs=None):
         As of now, only supports slices that preserve dimensionality.
         """
         if new_mgr_locs is None:
-            if isinstance(slicer, tuple):
-                axis0_slicer = slicer[0]
-            else:
-                axis0_slicer = slicer
```

Comment:
i tend to like this version because i can see in coverage output whether both cases are reached

Reply:
@jbrockmendel I can see your point, I like the one-liner version because it's making less noise (IMO). 

Anyway I am fine with reverting this one and also others.

Can we open a discussion for it (in a separate issue)? so we will put it in [Pandas code style guide](https://dev.pandas.io/docs/development/code_style.html)

Reply:
@jbrockmendel are you sticking on this one? +/- 0 on this.

Reply:
not a deal breaker